### ISSUE TITLE
FID metric

### ISSUE THREAD

**Issue by mxkai** [yiranran/APDrawingGAN#17]
Hi, thanks a lot for your awesome work.
I tried to compare FID metric between the real APDrawings in the training set and the test set, and the result is 43.2385, which is lower than the value in the paper. Can you provide the evaluation code used in the paper?

**Comment by yiranran** [comment]
We used this FID code: https://github.com/bioinf-jku/TTUR
